apps/src/lab2/views/components/Instructions.tsx

```diff
@@ -172,6 +172,36 @@ const InstructionsPanel: React.FunctionComponent<InstructionsPanelProps> = ({
   // the unique index.
   const useMessageIndex = useSecondaryFinishButton ? undefined : messageIndex;
 
+  const feedbackRef = useRef<HTMLDivElement>(null);
+  const runButton = document.querySelector('#run-button');
+
+  useEffect(() => {
+    const checkRunButton = () => {
+      if (feedbackRef.current && runButton?.textContent === 'Run') {
+        feedbackRef.current.focus();
```

Comment:
for my understanding: is focusing the ref what makes it be read aloud?

Reply:
Yes- exactly!